modules/javafx.graphics/src/main/java/javafx/scene/input/InputMethodRequests.java

```diff
@@ -51,6 +51,19 @@ public interface InputMethodRequests {
      */
     Point2D getTextLocation(int offset);
 
+    /**
+     * Gets the location of a specified offset in the current composed text,
+     * or of the selection in committed text. This information is, for example,
+     * used to position the candidate window near the composed text, or a
+     * composition window near the location where committed text will be
+     * inserted.
+     *
+     * @param offset the offset within the composed text, if there is
```

Comment:
how can an int be null?

Reply:
The return type is Point2D, not int.

But I'm glad you commented on it. I hadn't looked at this PR at all. Adding a method to a public interface is an API change, so at a minimum needs a CSR. More problematic is that since this isn't a default method, it is an incompatible change -- at least if any class outside of the JavaFX runtime were to implement it.

Reply:
I copied it from `getTextLocation`, but you're right.

Reply:
the null is in the @param block

Reply:
Ah, I see. Oops. I suspect it was intended on the `@return`, but now I'm not sure.